### DISCUSSION TITLE
Translate using Weblate

### DISCUSSION THREAD

**Discussion by Ionys320** [cachethq/cachet#4379] [Ideas]
Hi!
I wonder if Weblate would be a good platform to manage translations for Cachet (v3). The cloud-hosted version is free for open-source projects such as Cachet, and it could be a great way to handle modifications & cie. You can check [Weblate website here](https://weblate.org/)!

**Comment by jbrooksuk** [comment]
Hey @Ionys320,

Thanks for the suggestion! Before 3.x we were using [CrowdIn](https://crowdin.com), which worked really well for us - it has GitHub PR support, suggestions, Laravel support etc.

I'm not sure we _need_ to consider an alternative right now, but if, when I enable CrowdIn for 3.x, it's not right then I will take a look at Weblate.

**Reply by comradekingu** [reply]
Good stuff. Weblate is a better tool, and a lot better for privacy.
